src/color-variants.css

```diff
@@ -1579,57 +1659,77 @@ input:checked + .switch--dot-transparent::after {
  * @memberof Background colors
  * @example
  * <div class='grid'>
```

Comment:
Consider adding txt-s to this container so their aren't so many line breaks in the example:

<img width="744" alt="Screen Shot 2021-04-15 at 10 24 43 AM" src="https://user-images.githubusercontent.com/108094/114912442-d8bfad00-9dd4-11eb-8aba-1eaefe9d649a.png">

---

<img width="767" alt="Screen Shot 2021-04-15 at 10 24 37 AM" src="https://user-images.githubusercontent.com/108094/114912495-e7a65f80-9dd4-11eb-8617-645ef67c74ef.png">